AppIntegrity/README.md

```diff
@@ -0,0 +1,53 @@
+#  App Integrity
+
+## Package description
+
+This package contains the logic to implement app integrity checks using [Firebase App Check](https://firebase.google.com/docs/app-check). It therefore encorporates the Firebase SDK as a dependency, along with the Networking module to enable backend requests.
+
+## Overview
+
+### What is app integrity?
+
+App Integrity describes checks made (in this case, by Firebase AppCheck) to verify that a genuine instance of your app is accessing your backend resources and that the device accessing it has not been tampered with.
+
+### Why do we need it for mobile?
+
+We have implemented app integrity checks when the `/token` endpoint is called to ensure that the client is:
+1. genuine: the client is not being impersonated
+2. unmodified: the code we have deployed has not been changed by the device
+
+This therefore adds a layer of security to our authorization flow.
+
```

Comment:
@Magdaline-A Please may we amend this part to:

We have implemented app integrity checks to ensure that the mobile app is:

1. Genuine: the client is not being impersonated
2. Unmodified: The code we have deployed has not been changed by the device